docs/functions/Users/New-VCFUser.md

```diff
@@ -0,0 +1,53 @@
+# New-VCFUser
+
+### Synopsis
+Connects to the specified SDDC Manager and adds a new user.
+
+### Syntax
+```
+New-VCFUser -user <UPN String> -role <string>
+```
+
+### Description
+The New-VCFUser cmdlet connects to the specified SDDC Manager and adds a new user.
+
+### Examples
+#### Example 1
+```
+New-VCFUser -user sec-admin@rainpole.io -role ADMIN
```

Comment:
Should we consider using a different user example for New-VCFUser rather than the sec-admin role or at least have different between New-VCFUser and New-VCFServiceUser